client/lib/cart/store/cart-analytics.js

```diff
@@ -17,6 +17,8 @@ function recordEvents( previousCart, nextCart ) {
 
 function recordAddEvent( cartItem ) {
 	analytics.tracks.recordEvent( 'calypso_cart_product_add', cartItem );
+	analytics.ga.recordEvent( 'Checkout', 'calypso_cart_product_add',
+		`${ cartItem.product_id } ${ cartItem.product_slug }` );
```

Comment:
Yeah, I think the last argument should not be used for arbitrary number as it is used for some counts on GA dashboard.